storage/src/main/java/org/apache/kafka/server/log/remote/storage/RemoteLogManagerConfig.java

```diff
@@ -197,7 +197,7 @@ public final class RemoteLogManagerConfig {
                                   atLeast(0),
                                   LOW,
                                   REMOTE_LOG_METADATA_CUSTOM_METADATA_MAX_BYTES_DOC)
-                  .defineInternal(REMOTE_LOG_INDEX_FILE_CACHE_TOTAL_SIZE_BYTES_PROP,
+                  .define(REMOTE_LOG_INDEX_FILE_CACHE_TOTAL_SIZE_BYTES_PROP,
```

Comment:
Filed https://issues.apache.org/jira/browse/KAFKA-15535 to followup on adding documentation.

Reply:
Thanks, I have assigned it to myself.

Reply:
This configuration name and the corresponding doc have been provided by you before, but have not been used. This time, the configuration name is directly reused.
I also read other configuration docs related to tiered storage and found nothing missing. @satishd